src/csync/csync_update.cpp

```diff
@@ -668,6 +709,22 @@ int csync_ftw(CSYNC *ctx, const char *uri, csync_walker_fn fn,
         fullpath = QByteArray() % uri % '/' % filename;
     }
 
+    // When encountering placeholder files, read the relevant
+    // entry from the db instead.
+    if (ctx->current == LOCAL_REPLICA
+            && dirent->type == ItemTypeFile
+            && filename.endsWith(".owncloud")) {
```

Comment:
We will want to make this name configurable (in the theme)

Reply:
Suggestion: simply use the shortname (maybe add a more semantic getter to the theme but please do not introduce a whole new configurable var, we have too many of those anyway)